README.md

```diff
@@ -184,7 +184,9 @@ Prints:
 ## CLI
 
 ```bash
-jv [-draft INT] [-output FORMAT] <json-schema> [<json-doc>]...
+jv [-draft INT] [-output FORMAT] [-assert-format] <json-schema> [<json-doc>]...
```

Comment:
change it to `-assertformat`.
golang flag name generally contain single word (see go help run)

Reply:
Done.